examples/adafruit_io_simpletest_randomizer.py

```diff
@@ -55,9 +55,14 @@
 random_data_id = 1234
 
 while True:
-    print('Fetching random data from Adafruit IO...')
-    random_data = io.receive_random_data(random_data_id)
-    print('Random Data: ', random_data['value'])
-    print('Data Seed: ', random_data['seed'])
-    print('Waiting 1 minute to fetch new randomized data...')
-    time.sleep(60)
+    try:
+        print('Fetching random data from Adafruit IO...')
+        random_data = io.receive_random_data(random_data_id)
+        print('Random Data: ', random_data['value'])
+        print('Data Seed: ', random_data['seed'])
+        print('Waiting 1 minute to fetch new randomized data...')
```

Comment:
It looks like the `sleep(60)` was taken out so this line isn't quite true anymore.